programs/mcperf/meta.yaml

```diff
@@ -0,0 +1,17 @@
+short_description: >
+ twemperf (pronounced "two-em-perf"), aka mcperf is a tool for measuring memcached server performance. mcperf is like httperf, but for memcached protocol. It speaks memcached ASCII protocol and is capable of generating connections and requests at a high rate.
+
+description: >
```

Comment:
use "mcperf is a tool for measuring memcached server performance. mcperf is like httperf, but for memcached protocol. It speaks memcached ASCII protocol and is capable of generating connections and requests at a high rate."

Reply:
ok